src/components/EventPage/RsvpSection.tsx

```diff
@@ -155,6 +157,7 @@ export const SessionsWrapper = ({
             {Intl.DateTimeFormat().resolvedOptions().timeZone}
           </span>
         </div>
+        <SessionsDetailsNonSubmittable sessions={sessions} />
```

Comment:
this needs to be inside a check that checks for currently logged in user. if the user is signed out the component should not be visible. then it needs to check if the user has already RSVPd to one of the events in the current convo. only then the component should be visible

Reply:
and if the checks pass, this component should replace the above component with an option to remove RSVP if the user wants